pkg/lint/lintersdb/enabled_set.go

```diff
@@ -77,16 +77,22 @@ func (es EnabledSet) build(lcfg *config.Linters, enabledByDefaultLinters []*lint
 	return resultLintersSet
 }
 
-func (es EnabledSet) Get(optimize bool) ([]*linter.Config, error) {
+func (es EnabledSet) GetEnabledLintersMap() (map[string]*linter.Config, error) {
+	if err := es.v.validateEnabledDisabledLintersConfig(&es.cfg.Linters); err != nil {
+		return nil, err
+	}
+
+	return es.build(&es.cfg.Linters, es.m.GetAllEnabledByDefaultLinters()), nil
+}
+
+func (es EnabledSet) GetOptimizedLinters() ([]*linter.Config, error) {
```

Comment:
Please leave a comment on this method, it is not fully clear what `Optimized` means on first look.

Reply:
thank you, I've left a comment